tests/tests++.cc

```diff
@@ -96,7 +96,7 @@ int main(int argc, char** argv) {
 
         print_test_summary(tgt_tests, passed, sir_timer_elapsed(&timer));
 
-        if (passed != tgt_tests) {
+        if (passed != tgt_tests) { // GCOVR_EXCL_START
```

Comment:
Artificially set passed to passed -1 when `--coverage` is on the CL?

Reply:
Maybe, but I'm not concerned enough to change the test harness here just because that code isn't getting tested. (And we know it works, and we've seen it "work" when we had a failing test.)